telemetry/definitions/commonDefinitions.json

```diff
@@ -1067,6 +1067,50 @@
                 { "type": "ec2ConnectionType" }
             ]
         },
+        {
+            "name": "ec2_createAmi",
+            "description": "Create an image from an EC2 Instance",
+            "metadata": [
+                { "type": "result" }
+            ]
+        },
+        {
+            "name": "ec2_editInstanceElasticIp",
+            "description": "Associate or disassociate an Elastic IP with an EC2 Instance",
+            "metadata": [
+                { "type": "result" }, 
+                { "type": "enabled", "required": false}
+            ]
+        },
+        {
+            "name": "ec2_editInstanceShutdownBehavior",
+            "description": "Adjust the shutdown behavior of an EC2 Instance",
+            "metadata": [
+                { "type": "result" }
```

Comment:
We'll probably move `result` to be a common field soon. There's little reason why we _wouldn't_ want it considering that every event could potentially fail.

Reply:
I was thinking the same thing today. I'm tempted to update the .NET generator to do this (much like #408), but would like to get these metrics hooked up to the Toolkit first.